cove_ocds/views.py

```diff
@@ -159,5 +159,14 @@ def explore_ocds(request, pk):
             context['releases'] = json_data['releases']
         else:
             context['releases'] = []
-
-    return render(request, template, context)
+    
+    # Check the path to know whether a 'raw' request has been made
+    request_path = request.path
+    if 'raw' in request_path.split('/'):
+        response = {}
+        response['validation_errors'] = context['validation_errors']
+        response['validation_errors_count'] = context['validation_errors_count']
+        # Output as raw json
```

Comment:
I think we should probably output the whole output of ```ocds_json_output``` at _cove/cove_ocds/lib/api.py_. Given that we don't have real api endpoints, we could give a response with all we've got and let the requester take from it what they need.

Please note that ```ocds_json_output``` validation errors are output a bit differently to the one in the view ```context```, so this ```context['validation_errors']```:

```
[('["date-time", "Date is not in the correct format", "publishedDate"]', [{'value': '2014-11-07', 'path': 'publishedDate'}]), ('["date-time", "Date is not in the correct format", "releases/date"]', [{'value': '2014-03-25', 'path': 'releases/0/date'}, {'value': '2014-04-04', 'path': 'releases/1/date'}]), ('["date-time", "Date is not in the correct format", "releases/tender/tenderPeriod/endDate"]', [{'value': '2014-04-15 14:00 Eastern Daylight Time (EDT)', 'path': 'releases/0/tender/tenderPeriod/endDate'}, {'value': '2014-04-22 14:00 Central Daylight Time (CDT)', 'path': 'releases/1/tender/tenderPeriod/endDate'}]), ('["required", "\'id\' is missing but required", "releases"]', [{'path': 'releases/0'}]), ('["uri", "Invalid \'uri\' found", "publisher/uri"]', [{'value': 'buyandsell.gc.ca', 'path': 'publisher/uri'}])]
```
will look like this in the cli output version of it:
```
...
  "validation_errors": [
    {
      "value": "2014-11-07",
      "type": "date-time",
      "path": "publishedDate",
      "description": "Date is not in the correct format",
      "field": "publishedDate"
    },
    {
      "value": "2014-03-25",
      "type": "date-time",
      "path": "releases/0/date",
      "description": "Date is not in the correct format",
      "field": "releases/date"
    },
    {
      "value": "2014-04-04",
      "type": "date-time",
      "path": "releases/1/date",
      "description": "Date is not in the correct format",
      "field": "releases/date"
    },
    {
      "value": "2014-04-15 14:00 Eastern Daylight Time (EDT)",
      "type": "date-time",
      "path": "releases/0/tender/tenderPeriod/endDate",
      "description": "Date is not in the correct format",
      "field": "releases/tender/tenderPeriod/endDate"
    },
    {
      "value": "2014-04-22 14:00 Central Daylight Time (CDT)",
      "type": "date-time",
      "path": "releases/1/tender/tenderPeriod/endDate",
      "description": "Date is not in the correct format",
      "field": "releases/tender/tenderPeriod/endDate"
    },
    {
      "value": "",
      "type": "required",
      "path": "releases/0",
      "description": "'id' is missing but required",
      "field": "releases"
    },
    {
      "value": "buyandsell.gc.ca",
      "type": "uri",
      "path": "publisher/uri",
      "description": "Invalid 'uri' found",
      "field": "publisher/uri"
    }
  ]
...
```
And it won't include any aggregations, that will be up to the consumer of the output to do (it won't include a ``` 'validation_errors_count'``` either)

Hope this makes sense.

Reply:
Yes, this makes sense. Only issue has to do with accessing `ocds_json_output` from `explore_ocds`. I think I already tried to do that but couldn't retrieve the validation errors produced by `ocds_json_output`. That's why I resorted to pulling them from `context`.

Any suggestions?

Reply:
I think this would work:
```
from cove_ocds.lib.api import context_api_transform
... 
    response = context_api_transform(context)
...
```
Validation errors should be ```response['validation_errors']```